aspnetcore/fundamentals/host/index.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: Host in ASP.NET Core
+title: Web host and generic host in ASP.NET Core
```

Comment:
Proper nouns

Reply:
btw - If this goes this way, it will need to change again. Web Host is on the slide. It will take a little while, but it's not going to be the preferred host at 3.0. First, the names would perhaps be swapped in the title, then Web Host would be totally removed in some future release.